draft-ietf-taps-transport-security.md

```diff
@@ -463,10 +463,9 @@ sent in special INITIAL and CRYPTO frames {{I-D.ietf-quic-transport}}, types of
 are encrypted using different keys. INITIAL frames are encrypted using "fixed" keys derived
 from the QUIC version and public packet information (Connection ID). CRYPTO frames are
 encrypted using TLS handshake secrets. Once TLS completes, QUIC uses the resultant traffic
-secrets to for the QUIC connection to protect the rest of the streams. QUIC supports
-0-RTT (early) data using previously negotiated connection secrets. Early data is sent
-in 0-RTT packets, which may be included in the same datagram as the Initial and
-Handshake packets.
+secrets to protect the rest of the streams. QUIC supports
+0-RTT (early) data using previously negotiated connection secrets. (AF: please confirm the following changes are correct.) Early data is sent
+in 0-RTT packets, which may be included in the same datagram as the INITIAL and CRYPTO frames.
```

Comment:
I don't see an INITIAL frame type in transport-19, but I do still see an Initial packet type (0x0), and I also see 0-RTT is a packet type (0x1). Perhaps Tommy or Chris can suggest updated text here.

Reply:
Right, it's not an INITIAL frame, but an Initial packet. The text should be: ".. which may be included in the same UDP datagram as an Initial packet containing a CRYPTO frame".